shared/src/main/scala/com/cibo/evilplot/plot/renderers/BoxRenderer.scala

```diff
@@ -90,6 +99,39 @@ object BoxRenderer {
     }
   }
 
+  def tufte(
+    fillColor: Option[Color] = None,
+    strokeColor: Option[Color] = None,
+    lineDash: Option[LineDash] = None,
+    strokeWidth: Option[Double] = None
+  )(implicit theme: Theme): BoxRenderer = new BoxRenderer {
+
+    private val useFillColor = fillColor.getOrElse(theme.colors.fill)
+    private val useStrokeColor = strokeColor.getOrElse(theme.colors.path)
+    private val useLineDash = lineDash.getOrElse(theme.elements.lineDashStyle)
+    private val useStrokeWidth = strokeWidth.getOrElse(theme.elements.strokeWidth)
+
+    def render(plot: Plot, extent: Extent, context: BoxRenderer.BoxRendererContext): Drawable = {
+      val summary = context.summaryStatistics
+      val scale = extent.height / (summary.upperWhisker - summary.lowerWhisker)
+      val topWhisker = summary.upperWhisker - summary.upperQuantile
+      val uppperToMiddle = summary.upperQuantile - summary.middleQuantile
+      val middleToLower = summary.middleQuantile - summary.lowerQuantile
+      val bottomWhisker = summary.lowerQuantile - summary.lowerWhisker
+
+      Align
+        .center(
+          StrokeStyle(Line(scale * topWhisker, useStrokeWidth / 2), useStrokeColor).rotated(90),
```

Comment:
Discrete axes currently have a bit of extra padding, which mean they don't align well with these lines:
<img width="1085" alt="screen shot 2018-07-24 at 10 14 50 am" src="https://user-images.githubusercontent.com/1902485/43144264-b43bf7ba-8f2a-11e8-815c-82a2111761b5.png">

Shifting them so they are in the middle of their extent should resolve it, e.g. with `translate(extent.width / 2)` for each line.
<img width="1082" alt="screen shot 2018-07-24 at 10 18 25 am" src="https://user-images.githubusercontent.com/1902485/43144359-edff7daa-8f2a-11e8-9836-97c44a5936e9.png">

Reply:
Great, that was something I was looking for!